src/v2/dbus/manager.py

```diff
@@ -0,0 +1,93 @@
+import os
+from abc import ABC
+from abc import abstractmethod
+from gi.repository import Gio
+
+from hack_sound_server.utils.loggable import Logger
+from hack_sound_server.v2.utils.misc import dbus_method_to_signal
+from hack_sound_server.v2.utils.misc import snakecase
+
+
+class DBusManager(ABC):
```

Comment:
Because this is a helper to create a manager for the server, the sound and the player.
It also takes care of avoid the repeating code needed to handle dbus methods  on the different target objects.